doc/source/tune-searchalg.rst

```diff
@@ -155,6 +156,30 @@ An example of this can be found in `skopt_example.py <https://github.com/ray-pro
     :show-inheritance:
     :noindex:
 
+Ax Search
+---------
+
+The ``AxSearch`` is a SearchAlgorithm that is backed by `Ax <https://ax.dev/>`__ to perform sequential model-based hyperparameter optimization. Note that this class does not extend ``ray.tune.suggest.BasicVariantGenerator``, so you will not be able to use Tune's default variant generation/search space declaration when using SkOptSearch.
```

Comment:
maybe also include a one-line comment about what is Ax. `Ax is a  platform for understanding, managing, deploying, and automating adaptive experiments. Ax provides an easy to use interface with BoTorch, a flexible, modern library for Bayesian optimization in PyTorch.`

Reply:
Also says SkOptSearch here.